intgtest/README.md

```diff
@@ -17,10 +17,10 @@ to load the artifacts into the server:
 
 The following CLI commands will load these into the Vantiq server:
 
-    % vantiq -s <profile> load type      ./type/TestType.json
-    % vantiq -s <profile> load ruleset   ./rule/onTestPublish.vail
-    % vantiq -s <profile> load procedure ./procedure/echo.vail
-    % vantiq -s <profile> load source    ./sources/JSONPlaceholder.json
+    % vantiq -s <profile> load type      ./project/types/TestType.json
+    % vantiq -s <profile> load ruleset   ./project/rules/onTestPublish.vail
+    % vantiq -s <profile> load procedure ./project/procedures/echo.vail
+    % vantiq -s <profile> load source    ./project/sources/JSONPlaceholder.json
```

Comment:
I think these "load" commands pre-date the "import" command, and now these 4 lines can be replaces with a single "import"
```suggestion
    % vantiq -s <profile> import -d  ./project
```